src/test/java/com/github/stickerifier/stickerify/media/MediaHelperTest.java

```diff
@@ -181,6 +182,13 @@ void noAnimatedStickerConversionNeeded() throws Exception {
 		assertThat(result, is(nullValue()));
 	}
 
+	@Test
+	void isAnimationLowFpsCompliant() throws Exception{
+		var animatedSticker= resources.loadResource("low_fps_animated_sticker.tgs");
+		var result = MediaHelper.convert(animatedSticker);
+		assertThat(result,is(nullValue()));
+
+	}
```

Comment:
The test is overall ok, good job! I tested and without your fix it would break ✅ 
I only ask you to format it like the others in this class: 
- [ ] it's missing a space after `throws Exception`
- [ ] it's missing a space after `var animatedSticker`
- [ ] it's missing a space after `assertThat(result,`
- [ ] there should be no empty line before the closing bracket of the method
- [ ] there should be an empty line between a test and the other

Reply:
Done all the changes requested.

Reply:
You missed the space between `result,` and `is` 🕵🏻

Reply:
Done.